app/helpers/navbar_helper.rb

```diff
@@ -0,0 +1,20 @@
+module NavbarHelper
```

Comment:
The way we're using this module, i don't think it belongs in `app/helpers`, which have a specific usage pattern in rails. 
I'd consider moving this to `app/lib` and naming it `NavbarVisibility` or some such.

Reply:
Done and now i understand the difference between Helpers directory and lib ! :+1: